java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -111,6 +158,11 @@ public interface StringCommands {
      * @see <a href="https://redis.io/commands/decr/">redis.io</a> for details.
      * @param key The key to decrement its value.
      * @return The value of <code>key</code> after the decrement.
+     * @example
+     *     <pre>
+     * Long num = client.decr("key").get()
```

Comment:
Should we add examples with error scenarios? e.g. value is not a number

Reply:
I don't think we need to add every single scenario for these examples. They can check IT tests for extensive ones.